### PR TITLE
DOCS/FAQ: re-worded static build section

### PR DESCRIPTION
**Pull request by hoopoepg**
- re-worded some text in static build section

### CODE REVIEW

**Review by yosefe — APPROVED** [review]

**Comment by yosefe** [comment]
@tonycurtis @shamisp can you pls take a look?

File: docs/source/faq.md [hunk 1]
```diff
@@ -117,19 +117,19 @@ The file can be created using `ucx_info -Cf`.
...
 pkg-config can be incorporated in a Makefile-based build:
 ```
 program: program.c
-        cc program.c $(pkg-config --cflags --libs ucx)
+        $(CC) program.c $(shell pkg-config --cflags --libs ucx)
 ```
-When linking with static UCX libraries, need to list explicitly all required
-transport modules.  For example, in order to support only cma and knem
-transports, can use this method:
+When linking with static UCX libraries, the user needs to list all required
```
Comment (shamisp) [inline docs/source/faq.md:127]: needs -> must

File: docs/source/faq.md [hunk 2]
```diff
@@ -117,19 +117,19 @@ The file can be created using `ucx_info -Cf`.
...
+transport modules explicitly.  For example, in order to support only cma and
+knem transports, the user can use:
```
Comment (shamisp) [inline docs/source/faq.md:129]: have to use:

**Review by shamisp — CHANGES_REQUESTED** [review]
@tonycurtis  ?

**Review by tonycurtis — DISMISSED** [review]

File: docs/source/faq.md (hunk 1 above)
Comment (hoopoepg) [reply docs/source/faq.md]: updated

File: docs/source/faq.md (hunk 2 above)
Comment (hoopoepg) [reply docs/source/faq.md]: updated

**Review by shamisp — APPROVED** [review]

**Comment by hoopoepg** [comment]
@yosefe ok to merge?

**Comment by shamisp** [comment]
@hoopoepg it is good to go.

**Pull request merged**